### PR TITLE
Move DefaultSupportedLTS to juju

### PR DESCRIPTION
**Pull request by SimonRichardson**
Juju should be locked stepped with the default supported LTS and not
depend on an external package. This forces Juju to understand that when
a new LTS is updated and that we depend on that LTS then it should be
correctly updated when we do a release.

Unfortunately, we missed the default LTS for 2.8 and are instead doing it
in 2.9, with these changes, we should be in a better chance to spot that.

This PR does not update the default from bionic to focal, that will come
in a subsequent PR.

## QA steps

Only mechanical change from `juju/series` to `juju/juju`, so all tests should
pass.

### CODE REVIEW

File: environs/jujutest/livetests.go
Comment (manadart) [inline environs/jujutest/livetests.go:223]: Is this right? Diverges from `jujuversion` below.

**Review by manadart — APPROVED** [review]
I am not sure of wider implications, but it makes sense to me that Juju proper should state what it's default target Ubuntu version is, rather than having it in a library.

Should we remove it upstream too?

File: environs/jujutest/livetests.go
Comment (SimonRichardson) [reply environs/jujutest/livetests.go:223]: I'm waiting for CI tests to catch mistakes :cry:

**Comment by SimonRichardson** [comment]
> Should we remove it upstream too?

Once my download refinements #12101 lands, as that tweaks the series package, yes I'll remove it from upstream.

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**
